zulipterminal/ui_tools/boxes.py

```diff
@@ -1361,6 +1361,26 @@ def main_view(self) -> List[Any]:
         ]
         return [part for part, condition in parts if condition]
 
+    def update_msg_content_header(self) -> bool:
+        """
+        Update the content header of a message box (if required).
+        """
+        update_required = False
+
+        if self.need_recipient_header():
+            update_required = self[1][1].text != " "
```

Comment:
Could we store `self[1][1]` in a variable before so that it is more clear what is being checked?